font.js

```diff
@@ -1,9 +1,45 @@
 function setFontFamily(element, data) {
   const fontFamily = data.fontFamily;
-  console.log(fontFamily);
+
+  let existingStyle = document.querySelector(".accessorease-font-style");
+
+  if (existingStyle) {
+    existingStyle.remove();
+  }
+
+  if (fontFamily !== "Default") {
+    // Add style to body
+    const styleGlobal = document.createElement("style");
+    styleGlobal.innerHTML = `
+           * {
+               font-family: ${fontFamily} !important;
+           }
+       `;
+
+    styleGlobal.className = "accessorease-font-style";
+
+    document.head.appendChild(styleGlobal);
+  }
 }
 
 function setFontSize(element, data) {
   const fontSize = data.fontSize;
-  console.log(fontSize);
+
+  let existingFontSize = document.querySelector(".dynamic-font-size");
```

Comment:
Use ".accessorease-font-size" here instead of ".dynamic-font-size". This is to make sure the class belongs to our browser extension. The page may have a class called "dynamic-font-size", and if it does, there would be a chance the query selector will get the wrong element